build.gradle

```diff
@@ -743,7 +743,7 @@ subprojects {
   test.dependsOn('spotbugsMain')
 
   tasks.withType(com.github.spotbugs.snom.SpotBugsTask) {
```

Comment:
Could we address https://docs.gradle.org/current/userguide/task_configuration_avoidance.html in this PR?

`tasks.withType(com.github.spotbugs.snom.SpotBugsTask).configureEach`